doc/config-schemas/setAlarmUser.md

```diff
@@ -0,0 +1,48 @@
+# Set user To Alarm partition
+
+## Descripcion
+
+Agrega un usuario a una particion de alarma
+
+## Request Message
+
+```json
+{
+    ...
+    "data": {
+        "deviceData": {...},
+        "configKey": "setAlarmUser",
+        "value":{
+            "systemId":"xxx-xxxx",
+            "userName": "Em-32",
+            "userId:"xxx-xxxx",
+            "accessCode": "5125"
+        }
+    }
+}
+
```

Comment:
Why not here need the number propiertie in the request value, for example in setAlarmPartition the number is used in the value, why not here?